app/controllers/api/v1/game_play_apis_controller.rb

```diff
@@ -1,8 +1,18 @@
-# require 'pry'; binding.pry
+# require './././app/serializer/game_play_serializer'
 
 class Api::V1::GamePlayApisController < ApplicationController
   def index 
-    @word = WordFacade.find_word
-    # require 'pry'; binding.pry
+    @answers = WordFacade.multi_word_search(num_param)
+    @photo = PhotoFacade.photos(@answers[0])
+    render json: GamePlaySerializer.play_response(@photo, @answers[0], @answers), status:200
```

Comment:
Let's add a "to do" on the project board to ament the WordFacade response to a hash ...which would impact the arguments being passed to the GPS

Reply:
Added to project board